src/routes/auth.routes.js

```diff
@@ -6,13 +6,19 @@ import {
   login,
   changePassword,
   deleteUser,
+  requireAllfields,
+  validationErrors,
+  // eslint-disable-next-line import/named
+  requireCredentials,
```

Comment:
these aren't controllers, if you are passing it as middlewares move it to middleware

Reply:
ok , I will move them to middleware folder